phpunit.xml

```diff
@@ -12,6 +12,10 @@
          verbose="true"
          colors="true">
     <testsuites>
+        <testsuite name="Connection">
+            <file>./tests/components/DBServiceProviderTest.php</file>
+            <file>./tests/components/MySQLConnectionTest.php</file>
+        </testsuite>
```

Comment:
Testsuite includes all existing tests related to database connections. If any new `Connection` classes are to be added later then they also should be included in this testsuite.